test/OpenTelemetry.Tests/Internal/PeriodicExportingMetricReaderHelperTests.cs

```diff
@@ -33,6 +33,18 @@ public void CreatePeriodicExportingMetricReader_Defaults()
         Assert.Equal(MetricReaderTemporalityPreference.Cumulative, reader.TemporalityPreference);
     }
 
+    [Fact]
+    public void CreatePeriodicExportingMetricReader_Defaults_WithTask()
+    {
+#pragma warning disable CA2000 // Dispose objects before losing scope
+        var reader = CreatePeriodicExportingMetricReader(useThreads: false);
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
Why suppress rather than fix?

Reply:
Mostly because the same test above is doing it, but indeed, it can be disposed.